tasks/vendoring/vendor_passa.py

```diff
@@ -9,10 +9,11 @@
 def vendor_passa(ctx):
     with TemporaryDirectory(prefix='passa') as passa_dir:
         vendor_dir = _get_vendor_dir(ctx).absolute().as_posix()
+        log("Installing requirements for build...")
+        ctx.run("python -m pip install -e git+https://github.com/sarugaku/passa.git#egg=passa plette[validation] requirementslib distlib pip-shims invoke --exists-action=w")
         ctx.run("git clone https://github.com/sarugaku/passa.git {0}".format(passa_dir.name))
         with ctx.cd("{0}".format(passa_dir.name)):
             # ctx.run("git checkout 0.3.0")
-            ctx.run("pip install plette[validation] requirementslib distlib pip-shims -q --exists-action=i")
             log("Packing Passa")
             ctx.run("invoke pack")
```

Comment:
only need to pack passa w/o dependencies, which are already in vendors/patched dirs.

This will reduce the package size.

Reply:
please don't give feedback on my PR unless you understand why i am doing what i did or tried it yourself

Reply:
being that it runs in a zipfile and without access potentially to anything in pipenv's path, it doesn't really make any sense to trust the vendoring in pipenv